data/migrations/versions/d94d328733e4_increase_repomirrorconfig_char_size.py

```diff
@@ -0,0 +1,45 @@
+"""Increase repomirrorconfig char size
+
+Revision ID: d94d328733e4
+Revises: 909d725887d3
+Create Date: 2021-10-21 14:16:07.660766
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "d94d328733e4"
+down_revision = "909d725887d3"
```

Comment:
I think the `down_revision` needs to be `e9f3e4dbb979` instead.

Reply:
@flavianmissi I updated it with the most recent revision.